README.md

```diff
@@ -0,0 +1,15 @@
+# Install Spice from Brew
+
+## Pre-requisites
+
+Noto: when using linux homebrew, install build-essentials or equivalent package.
+
+## Install `spice`
+```
+brew install spiceai/spiceai/spice
```

Comment:
It's not possible to make this root, is it? So we can do `brew install spiceai/spiceai`?

Reply:
yeah would be nice if they allow so. But I haven't seen any doc about this.